.github/workflows/build.yml

```diff
@@ -190,6 +191,13 @@ jobs:
           cp -r ${{ env.INSTALL_DIR }} ${{ env.INSTALL_PORTABLE_DIR }}  # cmake install on Windows is slow, let's just copy instead
           cmake --install ${{ env.BUILD_DIR }} --prefix ${{ env.INSTALL_PORTABLE_DIR }} --component portable
 
+      - name: Package (Windows, installer)
+        if: runner.os == 'Windows'
+        shell: msys2 {0}
+        run: |
+          cd ${{ env.INSTALL_PORTABLE_DIR }}
```

Comment:
Why are you packaging the portable version into an installer. This would cause the data folder to be where PolyMC is installed. This should probably rather be the non-portable version

Reply:
At install time, there’s an optional checkbox that’s labeled “portable” which is deselected by default. The portable component is only installed if the user selects that checkbox; by default, it installs the non-portable version. I have to use the portable directory in order to get that file.

Reply:
Oh hmm interesting. What will be the target location if the user selects the portable version?

Reply:
It’s the same location by default. Although if they select the portable version, I’d assume they’d change the install location when asked to do so.